apisprout.go

```diff
@@ -607,6 +608,21 @@ func server(cmd *cobra.Command, args []string) {
 		w.Write(encoded)
 	})
 
+	// Another custom handler to return the exact swagger document given to us
+	http.HandleFunc("/_sprout/schema", func(w http.ResponseWriter, req *http.Request) {
```

Comment:
Can we continue using the  double underscore `__` prefix? We have `__reload` and `__health` already, so this can be `__schema` which should have a low chance of shadowing. Otherwise this looks great!

Reply:
Doneski